src/main/java/emis/Parser.java

```diff
@@ -0,0 +1,64 @@
+package main.java.emis;
+import main.java.emis.exceptions.EmisException;
+import main.java.emis.command.*;
+import main.java.emis.task.*;
+
+// Parser class, deals with making sense of the user command
+public class Parser {
+    public static Command parse(String fullCommand) throws EmisException {
+        Integer spaceIndex = fullCommand.indexOf(" ");
+
+        if (spaceIndex == -1) {
+            // no spaces found
+            // commands are either "bye", "list", or invalid command
+            if (fullCommand.equals("bye")) {
+                return new ExitCommand();
+            } else if (fullCommand.equals("list")) {
+                return new PrintCommand();
+            } else {
+                throw new EmisException("Invalid command.");
+            }
+        } else {
+            // spaces found
+            // possible commands are "todo", "deadline", "event", "mark", "unmark", "delete"
```

Comment:
an enum here would be really useful!

`enum Actions {
	TODO,
	DEADLINE,
	EVENT
}`